src/main/webapp/creatives/maps/simid_map_creative.js

```diff
@@ -42,6 +43,7 @@ export default class SimidMapCreative extends BaseSimidCreative {
   specifyButtonFeatures_(buttonLabel = DEFAULT_BUTTON_LABEL) {
     const findNearestButton = document.getElementById('findNearest');
     findNearestButton.innerText = FIND_NEAREST_TEMPLATE_TEXT + buttonLabel;
+
```

Comment:
I just wouldn't add a newline unless you think it needs it.

Reply:
Must have been extra from when I deleted the console.log. Got rid of it